docker/dockerfiles/zkv-relay.Dockerfile

```diff
@@ -36,7 +36,7 @@ ARG PROFILE="release"
 ARG FEATURES=""
 
 ENV BINARY="${BINARY}" \
-  RUN_USER=hl
+  RUN_USER=user
```

Comment:
That's ok.... but, why did you change it? I spent lot of time to catch the best one :smile: 
`hl` stand for horizen labs

Reply:
zkVerify has been moved to zkVerify org and EDS is the one working on the development. I think we should get rid of everything "hl".